src/main/kotlin/com/cout970/magneticraft/api/internal/registries/machines/kiln/KilnRecipe.kt

```diff
@@ -0,0 +1,47 @@
+package com.cout970.magneticraft.api.internal.registries.machines.hydraulicpress
+
+import com.cout970.magneticraft.api.internal.ApiUtils
+import com.cout970.magneticraft.api.registries.machines.kiln.IKilnRecipe
+import net.minecraft.block.state.IBlockState
+import net.minecraft.item.Item
+import net.minecraft.item.ItemStack
+import net.minecraftforge.oredict.OreDictionary
+
+/**
+ * Created by cout970 on 22/08/2016.
+ */
+data class KilnRecipe(
+        private val input: ItemStack,
+        private val itemOutput: ItemStack?,
+        private val blockOutput: IBlockState?,
```

Comment:
There should be 2 classes for Kiln recipes, one using items as output and other using a block as ouput